### ISSUE TITLE
Global properties are broken

### ISSUE THREAD

**Issue by rtollert** [gentoo/eudev#142]
e.g. if there's some rule involving ENV{foo}="bar", `udevadm control --property=foo=bar` does *not* satisfy that condition in eudev 3.1.5.

This is a regression against udev 182, and in fact, is almost certainly a mismerge against systemd: The systemd commit adeba5008eac3105ae59256dedd087ebe006a9e6 "udev: support ENV{}=="" global property matches" was never applied to eudev.

(The preceding commit, 41b848b0ea674dba135d5d5f82b87d126d967fcf "udev: move global property handling from libudev to udevd", was applied as eudev commit 5fe138cdda4eb51036062b4bddce258fe97e7b68, albeit with many merge changes seemingly unannounced in the commit message. It looks like 41b848b0ea674dba135d5d5f82b87d126d967fcf more or less removed the actual logic behind global properties and adeba5008eac3105ae59256dedd087ebe006a9e6 added it back in a refactored way.)

So, basically, systemd commit adeba5008eac3105ae59256dedd087ebe006a9e6 needs to get merged.

**Comment by blueness** [comment]
sorry for missing this, i've been busy with real life.  i agree and will try to have that in soon.  if you want to propose a patch against master/HEAD i'd be happy to review and commit.

**Comment by blueness** [comment]
Okay I see what happened here.  I must have missed a hunk when merging adeba500.  This should be fixed now.  Please reopen this if its still an issue.
